### PR TITLE
Automatically focus the ColorSchemeListView when we navigate to the ColorSchemes page

### LINKED ISSUE #11971: [Settings>Color schemes] : Keyboard focus jumps to first tab, when user activates 'Yes, delete color scheme' button.
### Windows Terminal version

1.12.3472.0

### Windows build number

22509.1011

### Other Software

**Test Environment: **
OS: Windows 11 Version Dev (OS Build 22509.1011)
App: Windows Terminal Preview

### Steps to reproduce

**Repro Steps: **

1.Open Windows Terminal.
2.Open Settings page using 'Ctr+,'
3.Navigate to 'Color schemes' list-item present in left pane via tab ley & press enter.
4.Naviagte to 'Delete color scheme' button via tab key & press enter.
5.A pop-up will gets open.
6.Navigate to 'Yes, delete color scheme' button via tab key & press enter.
7.Observe the issue.

**User Experience: **
Users who rely on keyboard will get impacted here & will have to navigate again from top of the page un-necessarily, if Keyboard focus jumps to first tab on activating 'Yes, delete color scheme' button.

Guideline reference
https://www.w3.org/TR/WCAG21/#focus-order

Attachment : 
[Keyboard focus jumps to first tab, when user activates 'Yes, delete color scheme' button..zip](https://github.com/microsoft/terminal/files/7728455/Keyboard.focus.jumps.to.first.tab.when.user.activates.Yes.delete.color.scheme.button.zip)

### Expected Behavior

Focus should land on next interactive control i.e (Discard changes button), on activating 'Yes, delete color scheme' button.

### Actual Behavior

Keyboard focus jumps to first tab, when user activates 'Yes, delete color scheme' button.

**Note :** 
1.Same issue repro on 'Deleting' profiles also.

### PR DESCRIPTION
**Pull request by PankajBhojwani**
This regressed when we implemented ColorSchemeViewModel in #13179.
This commit fixes that by automatically focusing the ColorSchemeListView
when we navigate to the ColorSchemes page, which makes sense from a
user experience perspective anyway.

Closes #11971

### CODE REVIEW

**Review by carlos-zamora — APPROVED** [review]
Easy Peasy Billy Butcher GIF

File: src/cascadia/TerminalSettingsEditor/ColorSchemes.h [hunk 1]
```diff
@@ -25,6 +26,9 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         WINRT_OBSERVABLE_PROPERTY(Editor::ColorSchemesPageViewModel, ViewModel, _PropertyChangedHandlers, nullptr);
 
         WINRT_CALLBACK(PropertyChanged, Windows::UI::Xaml::Data::PropertyChangedEventHandler);
+
+    private:
+        winrt::Windows::UI::Xaml::Controls::SwapChainPanel::LayoutUpdated_revoker _layoutUpdatedRevoker;
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsEditor/ColorSchemes.h:31]: probably you don't want to use the SwapChainPanel version of this ;P

Comment (DHowett) [reply src/cascadia/TerminalSettingsEditor/ColorSchemes.h:31]: It doesn't actually matter, though.

```suggestion
        winrt::Windows::UI::Xaml::FrameworkElement::LayoutUpdated_revoker _layoutUpdatedRevoker;
```

this _should_ work!

Comment (PankajBhojwani) [reply src/cascadia/TerminalSettingsEditor/ColorSchemes.h]: Fixed! Thanks

**Review by zadjii-msft — APPROVED** [review]
Is there no way to say in XAML "this is the thing that should start with focus"?

**Review by DHowett — APPROVED** [review]

**Pull request merged**